common/components/test/utils.test.js

```diff
@@ -79,7 +79,11 @@ describe('utils', () => {
 
   	const args = urlHelper.arguments('/api/test?query=test&page=1');
   	expect(args['query']).toEqual('test');
-  	expect(args['page']).toEqual('1');
+    expect(args['page']).toEqual('1');
+    
+    expect(urlHelper.isEmptyStringOrValidUrl('')).toEqual(true);
+    expect(urlHelper.isEmptyStringOrValidUrl('google.com')).toEqual(true);
+    expect(urlHelper.isEmptyStringOrValidUrl('localhost:3000')).toEqual(false);
```

Comment:
Here are some others to try, based on what we've seen people input into the links field.
### True ###
http://www.unsecure.com
https://www.secure.com
https://multiple.parts.com/subdir
https://hyphenated-domain.next.com/
https://www.gnarly.com/url/viewer.html?&something=d38d408127d64407a7627f8e990908fe&view=388155.63,109.56,-69891.37,388028.5,247.89,-70066.36,0.95&lyr=1,1,1,1,1,1,1,1,0,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1&wkid=2926&v=2
### False ###
N/A
TBD
Not built yet

Reply:
I have added these test cases to the validator tests.